src/home-page/components/CategoryCard.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Card, CardHeader, Avatar } from '@material-ui/core';
+import styled from 'styled-components';
+
+const CategoryCardContainer = styled(Card)`
+  width: 100%;
+  display: flex;
+  align-items: center;
+
+ :hover {
+   opacity: 0.8;
+ }   
+`;
+
+const CardHeaderContainer = styled(CardHeader)`
+ width: 100%;
+`;
+
+const AvatarContainer = styled(Avatar)<{backgroundColor: string}>`
+ margin-right: 1em;
+ background-color: ${props => props.backgroundColor};
+`;
+
+type CardProps = {
+  title: string;
+  description: string;
+  avatarSrc: string;
+  avatarBackground: string;
+};
+
+export const CategoryCard: React.FC<CardProps> = ({
+  title,
+  description,
+  avatarSrc,
+  avatarBackground,
+}) => {
+  return (
+    <CategoryCardContainer
+      onClick={() => {
```

Comment:
We should probably also add `onClick` to `CardProps` - we will need it to pass the click handler function from the parent component :)

Reply:
Done ✅